examples/ELZ-ReferralServiceRequest-01.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ServiceRequest xmlns="http://hl7.org/fhir">
+    <id value="ELZ-ReferralServiceRequest-01"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ELZ-ReferralServiceRequest"/>
+        <tag>
```

Comment:
This tag should be removed, as it currently is not used (the corresponding section has also been removed from the TO).

Reply:
Done.